client/jetpack-onboarding/steps/site-title.jsx

```diff
@@ -78,10 +78,15 @@ class JetpackOnboardingSiteTitleStep extends React.PureComponent {
 							blogname={ this.state.blogname }
 							blogdescription={ this.state.blogdescription }
 							disabled={ isRequestingSettings }
+							isTitleRequired={ true }
```

Comment:
Let's shorten this to just `isTitleRequired` :slightly_smiling_face: JSX best practice FTW!